packages/core/src/sass/_globals.scss

```diff
@@ -562,7 +568,7 @@ $AU-border-radius: 3px !default;
  *
  * @return {integer}             - The contrast ratio
  */
-@function AU-color-contrast( $forground, $background ) {
+@function AU-color-contrast( $forground, $background, $silent: false ) {
```

Comment:
uh very good solution! I didn't think of that

Reply:
Yeah, didn't want the warnings to go away for good 😄